theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -37,9 +37,9 @@ def find_changed_packages(diff_range) {
 def find_changed_debs(diff_range) {
     def changed_debs = []
 
-    def changed_dependencies = find_added_or_changed_files(diff_range, 'dependencies/*/*/changelog').split()
-    def changed_plugins = find_added_or_changed_files(diff_range, 'plugins/*/changelog').split()
-    def changed_core = find_added_or_changed_files(diff_range, 'debian/*/*/changelog').split()
+    def changed_dependencies = find_added_or_changed_files(diff_range, 'dependencies/*/*').split()
```

Comment:
I think at this point, you can just drop the `*/*` at the end and make it `find_added_or_changed_files(diff_range, 'dependencies/*/*')`?

But hah, nope, the whole logic (below) relies on the fact that this returns ONE result per package, but now it would probably return multiple (when e.g. changelog and rules files change)